contracts/contracts/reward-contract/RewardByBlock.sol

```diff
@@ -39,10 +41,25 @@ contract RewardByBlock is EternalStorage, IRewardByBlock {
         _;
     }
 
+    function initialize(uint256 _blockRewardAmount, address[] calldata _homeBridgeList) external {
+      require(!initialized, "Can not be initiated twice!");
+      require(_blockRewardAmount > 0, "Block reward amount must be positive!");
```

Comment:
Is there anything wrong with a block reward of zero? Don't see a reason why we'd have to prevent this.

Reply:
Removed :heavy_check_mark: